charts/krane/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
Can if done in another PR ... but might want to consider adding a linter to the helm chart

```yaml
   helm:
      runs-on: ubuntu-latest
      steps:
        - name: Checkout
          uses: actions/checkout@v3
          with:
            fetch-depth: 0
        - name: Set up Helm
          uses: azure/setup-helm@v1
          with:
            version: v3.8.1
        - uses: actions/setup-python@v2
          with:
            python-version: 3.7
        - name: Set up chart-testing
          uses: helm/chart-testing-action@v2.2.1
        - name: Run chart-testing (lint)
         if: github.ref == 'refs/heads/master'
          run: ct lint
        - name: Run chart-testing (lint)
         if: github.ref != 'refs/heads/master'
          run: ct lint --check-version-increment=false
```

It will require you add maintainer links to the chart btw

Reply:
Good shout. Added in this PR.